sr_example/scripts/sr_example/sr_mix_examples_unsafe.py

```diff
@@ -10,6 +10,14 @@
 hand_commander = SrHandCommander()
 arm_commander = SrArmCommander()
 
+rospy.loginfo("Set arm teach mode ON")
+arm_commander.set_teach_mode(True)
+# sleep for some time during which the arm can be moved around by pushing it
+# but be careful to get away before the time runs out. You are warned
+rospy.sleep(20.0)
```

Comment:
This sleep should probably be in the library directly instead of the example. This way we make sure people are doing the sleep.